src/Components/cards/card.jsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
```

Comment:
![image](https://user-images.githubusercontent.com/67889207/168415334-0a4fb6f7-27f9-4b08-9e1e-fe3746a220e6.png)
Your cards are overflowing from containing them in a div should fix the issue..